packages/dds/matrix/src/matrix.ts

```diff
@@ -59,6 +63,57 @@ interface ISetOpMetadata {
 	localSeq: number;
 	rowsRefSeq: number;
 	colsRefSeq: number;
+	referenceSeqNumber: number;
+	clientId: string | undefined;
+}
+
+interface ISetCellChangePolicyOp {
+	type: MatrixOp.changeSetCellPolicy;
+}
+
+/**
+ * Events emiited by Shared Matrix.
+ * @public
+ */
+export interface ISharedMatrixEvents<T> extends ISharedObjectEvents {
+	/**
+	 * Emitted when there is a conflict when this client tries to set a cell value in the {@link SharedMatrix}.
+	 * It could be due to race condition between different clients where this client set op loses or when
+	 * the client tries to set again before its previous set is acked.
+	 * Race Condition: It means we did not know about the Set op from other client when we sent the set op, but
+	 * we received the remote Op and set that value for the cell. Now when we get our local op back, we can not
+	 * overwrite the value due to FWW and need to resolve conflict.
```

Comment:
I'd suggest two minor changes to this text:
1. Mention that such conflicts could happen only when FWW is used.
2. Talk in context of global state, not this client. Just mention that when two clients race and send changes without observing each other changes, then changes that gets sequenced last would be rejected, and only client who's changes rejected would be notified via this event, with expectation that it will merge its changes back by accounting new information (state from winner of the race)